packages/rest/src/__tests__/integration/rest.server.integration.ts

```diff
@@ -36,20 +36,39 @@ const FIXTURES = path.resolve(__dirname, '../../../fixtures');
 const ASSETS = path.resolve(FIXTURES, 'assets');
 
 describe('RestServer (integration)', () => {
-  it('exports url property', async () => {
-    const server = await givenAServer();
-    server.handler(dummyRequestHandler);
-    expect(server.url).to.be.undefined();
-    await server.start();
-    expect(server)
-      .to.have.property('url')
-      .which.is.a.String()
-      .match(/http|https\:\/\//);
-    await supertest(server.url)
-      .get('/')
-      .expect(200, 'Hello');
-    await server.stop();
-    expect(server.url).to.be.undefined();
+  describe('url', () => {
+    it('exports url property', async () => {
+      const server = await givenAServer();
+      server.handler(dummyRequestHandler);
+      expect(server.url).to.be.undefined();
+      await server.start();
+      expect(server)
+        .to.have.property('url')
+        .which.is.a.String()
+        .match(/http|https\:\/\//);
+      await supertest(server.url)
+        .get('/')
+        .expect(200, 'Hello');
+      await server.stop();
```

Comment:
When the test fails, this line is never called and the server will stay running until the test suite is finished. I am proposing to move this line to `afterEach` hook.

```ts
describe('RestServer (integration)', () => {
  let server: RestServer;

  afterEach('stop the server', async function teardown() {
    if (!server) return;
    await server.stop();
  });

  // the tests
});
```